mmcv/cnn/resnet.py

```diff
@@ -266,6 +265,7 @@ def __init__(self,
     def init_weights(self, pretrained=None):
         if isinstance(pretrained, str):
             logger = logging.getLogger()
+            from ..runner import load_checkpoint
```

Comment:
Shall we start to use BaseModule for resnet and vgg as examples?

Reply:
Maybe in a separated PR?